scipy/stats/tests/test_distributions.py

```diff
@@ -5297,6 +5297,17 @@ def test_sf_isf(self):
         x1 = stats.nakagami.isf(sf, nu)
         assert_allclose(x1, x0, rtol=1e-13)
 
+    @pytest.mark.parametrize('nu, loc, scale',
+                             itertools.product(np.linspace(0.5, 5, 4),
```

Comment:
If you parametrize w.r.t. each variable separately, e.g.:
```
@pytest.mark.parametrize('nu', ...)
@pytest.mark.parametrize('loc', ...)
...
```
I believe you get the same behavior without `itertools`.

That said, I'm not sure if we would want so many combinations to run every time a push is made to a PR, as we have limited CI resources to work with. Instead of all these, can you pick one or a few cases didn't work before your change and work after?

Reply:
You are right, I addressed your two points in 6a10aff0370833b176f14e34944a3735a1f65f6c.